packages/css/src/formElements/radiobutton/radiobutton.css

```diff
@@ -0,0 +1,76 @@
+.md-radiobutton {
+  font-family: 'Open sans';
+  font-size: 16px;
+  border: none;
+  padding: 0;
+}
+
+.md-radiobutton {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  position: relative;
+}
+
+.md-radiobutton:focus-within {
+  outline: 2px solid var(--mdPrimaryColor);
+  outline-offset: 2px;
+}
+
+.md-radiobutton:hover {
+  text-decoration: underline;
+}
+
+.md-radiobutton input[type='radio'] {
+  position: absolute;
+  left: 0;
+  opacity: 0;
+  cursor: pointer;
+}
+
+.md-radiobutton__check-area {
+  width: 1.45rem;
+  height: 1.45rem;
+  display: block;
+  background-color: #fff;
+  border: 1px solid var(--mdPrimaryColor);
+  border-radius: 50%;
+  margin: 0 0.5rem 0 0;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.md-radiobutton:not(.md-radiobutton--disabled):focus-within .md-radiobutton__check-area,
+.md-radiobutton:not(.md-radiobutton--disabled):hover .md-radiobutton__check-area {
+  background-color: var(--mdPrimaryColor20);
+  text-decoration: underline;
+}
+
+.md-radiobutton__selected-dot {
+  width: 0.6em;
+  height: 0.6em;
+  display: block;
+  background-color: var(--mdPrimaryColor);
+  border-radius: 50%;
+  margin: 0 auto;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  opacity: 0;
+  animation: fadeIn 0.3s ease-out 1 forwards;
+}
+
+/* Disabled */
+.md-radiobutton--disabled:hover {
```

Comment:
Keep in mind `:disabled` is a selector. This works with the `disabled` prop on an element. Any reason not to use this?

Reply:
Copy paste css from existing component. Avoiding refactoring unrelated things and follow existing structure.

Reply:
E.g prøver å ikke henge meg opp i måten det har blitt gjort tidligere - isåfall burde det refaktorers overalt det er relevant.